Patch: change the name of the sb1250 DUART from duart to ttyS
Needed for debian-installer
Status: the sb1250_duart.c driver needs a completely re-write...

diff --git a/drivers/char/sb1250_duart.c b/drivers/char/sb1250_duart.c
index 7819acc..57a227c 100644
--- a/drivers/char/sb1250_duart.c
+++ b/drivers/char/sb1250_duart.c
@@ -762,8 +762,8 @@ static int __init sb1250_duart_init(void
 		return -ENOMEM;
 
 	sb1250_duart_driver->owner = THIS_MODULE;
-	sb1250_duart_driver->name = "duart";
-	sb1250_duart_driver->devfs_name = "duart/";
+	sb1250_duart_driver->name = "ttyS";
+	sb1250_duart_driver->devfs_name = "tts/";
 	sb1250_duart_driver->major = TTY_MAJOR;
 	sb1250_duart_driver->minor_start = SB1250_DUART_MINOR_BASE;
 	sb1250_duart_driver->type            = TTY_DRIVER_TYPE_SERIAL;

